### PR TITLE
Make dropdown open

### PR DESCRIPTION
**Pull request by Xiaocao-Cxh**
Tell more about changes of codes.

### CODE REVIEW

**Comment by mamy-CS** [comment]
Test Comment

File: docs/quickstart/README.md [hunk 1]
```diff
@@ -450,16 +450,16 @@ This output represents the backend response. Now you should be able to make Torn
 
 ### Step 3b: Connecting to the Tornjak frontend to access the Tornjak UI
 
-Make sure that the backend is accessible from your browser at `http://localhost:10000`, as above, or the frontend will not work. 
+Make sure that the backend is accessible from your browser at `http://localhost:10000`, as above, or the frontend will not work.
 
 If you chose to deploy Tornjak with the UI, connecting to the UI is very simple. Otherwise, you can always run the UI locally and connect. See the two choices below:
 
 <details><summary><b> 🔴 [Click] Run the Tornjak frontend locally</b></summary>
```
Comment (maia-iyer) [inline docs/quickstart/README.md:457]: would you be able to make the same change in this line too?

**Review by maia-iyer — CHANGES_REQUESTED** [review]
Hello! nearly good, want to do the same for the other line in the file

**Review by maia-iyer — COMMENTED** [review]
LGTM! Thanks for doing this!

**Review by maia-iyer — APPROVED** [review]
Approving

**Pull request merged**